recipes/pydistcheck/meta.yaml

```
@@ -0,0 +1,42 @@
+{% set name = "pydistcheck" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pydistcheck-{{ version }}.tar.gz
+  sha256: 806b21446c4c544ec792e6313509d14735f8f93dd26f01d1ce8e5c556d79cb4d
+
+build:
+  skip: true  # [py<38]
```

```suggestion
  noarch: python
```
Isn't this a pure python package?

If you have Python requirements, you could set `- python >=3.8` in the `host/run` sections below.

I see that you have a `tomli  # [py<311]` requirement. This won't work with `noarch: python`. But would it really hurt to install tomli always, even if python is 3.11?

---

Yes, this is a pure Python package.

I'd hoped to have `tomli` be conditional on the Python version so that that package would be lighter-weight in later versions of Python, but I didn't understand that doing that meant imposing a larger burden on the `conda-forge` infrastructure.

I'm ok with it being unconditional if that means my little CLI here will be a bit less expensive for `conda-forge` to build and distribute.

I just pushed https://github.com/conda-forge/staged-recipes/pull/20913/commits/81c4af9abfea7bdd9468e171f31329c734685ac3 which modifies the recipe based on your suggestions:

* makes `tomli` an unconditional dependency
* switches to `noarch: python`
* puts a floor of `python >= 3.8` in `requirements.host` and `requirements.run`